python/cuml/common/base.pyx

```diff
@@ -164,16 +162,24 @@ class Base:
         del base  # optional!
     """
 
-    def __init__(self, handle=None, verbosity=logger.LEVEL_INFO,
-                 output_type=None, verbose=None):
+    def __init__(self, handle=None, verbose=True,
```

Comment:
verbose is True by default? Shouldn't it be False?

Reply:
Good catch